test/jest/fixtures.js

```diff
@@ -131,12 +137,74 @@ module.exports = function registerFixtures(global) {
     await test(playwright.toImpl);
   });
 
+  const allBrowsers = new Set();
+
+  process.on('SIGINT', () => {
+    const promises = [];
+    for (const browsers of allBrowsers) {
+      for (const browser of browsers)
+        promises.push(browser.close());
+    }
+    Promise.all(promises).then(() => process.exit(130));
+  });
+
   global.registerWorkerFixture('browserType', async ({playwright}, test) => {
-    await test(playwright[process.env.BROWSER || 'chromium']);
+    /** @type {BrowserType} */
+    const browserType = playwright[process.env.BROWSER || 'chromium'];
+    /** @type {Set<Browser|BrowserServer|BrowserContext} */
+    const browsers = new Set();
+    /** @type {BrowserType} */
+    const wrapper = {
```

Comment:
WDYT about using a proxy?

Reply:
Proxy doesn't actually help much here. It lets us avoid having executablePath and name methods, but those were easy. I was thinking it might help if someone adds a new launcher method. But a new launcher method will likely take launchOptions, and then I'd want someone to have to add a new wrapper method rather than have it work without the default browser options.